src/components/SingleComment.jsx

```diff
@@ -1,11 +1,38 @@
-const SingleComment = ({votes, author, body, created_at}) => {
+import { useContext, useState } from "react";
+import { UserContext } from "../contexts/User";
+import { deleteComment } from "../utils/api";
+
+const SingleComment = ({comment_id, votes, author, body, created_at}) => {
+    const {loggedInUser} = useContext(UserContext);
+    const [highlighted, setHighlighted] = useState(false);
+    const [deleteInfo, setDeleteInfo] = useState(false)
+    const [buttonDisable, setButtonDisable] = useState('')
+    
+    const handleDelete = () => {
+        setButtonDisable('disabled')
+        setDeleteInfo(true)
+        deleteComment(comment_id).then(() => { })
+        .catch((err) => {
+           setButtonDisable('')
```

Comment:
is there maybe a better data type we could use for buttonDisable instead of going between '' and 'disabled?